README.md

```diff
@@ -1,9 +1,86 @@
 # dgc-dotnet
-C# /.NET implementation of the [Electronic Health Certificate Specification](https://github.com/ehn-digital-green-development/ehn-dgc-schema) to create and decode Digital Green Certificates
+C# /.NET implementation of the [Electronic Health Certificate Specification](https://github.com/ehn-digital-green-development/ehn-dgc-schema) to create and decode Digital Covid Certificates
 
 ## About
-This is a dotnet standard library. 
-It's meant to be a library to be used within other dotnet projects
+This is a dotnet standard library.
+
+## How to use
+### Verifing a certificate
+```c#
+// First connect to the DCC Gateway
+// Get the TLS certificate 
+// From a certificate store or this could also be a file or some other methods
+X509Certificate2 cert = new X509Certificate2([ByteArrayOfTheCert])
+
+var gatewayService = new GatewayService("https://url to the gateway", cert);
+
+var dscs = gatewayService.GetAllDscFromGateway();
+
+// ISecretariatService needs to be implemented
+var secratariatService = new MyImpSecratariatService(dscs);
+
+// Decode the barcode to cwt object
+GreenCertificateDecoder decoder = new GreenCertificateDecoder();
+var cwt = decoder.Decode(barcodeContent);
+
+// Verify
+GreenCertificateVerifier verifier = new GreenCertificateVerifier(secratariatService);
+var (isvalid, reason) = await verifier.Verify(cwt);
+```
+
+### Issuing certificate
+
+```c#
+// Get the signing certificate with a private key
+// From a certificate store (HSM) or this could also be a file or some other methods
+X509Certificate2 cert = new X509Certificate2([ByteArrayOfTheCert])
+
+// Create CWT
+var ci = "URN:UVCI:01:IS:SomeUniqeID";
+
+CWT cwt = new CWT();
+cwt.DGCv1 = new DgCertificate
+{
+    Name = new Nam
+    {
+        FamilyName = lastName,
+        GivenName = firstName,
+        FamilyNameTransliterated = MrzEncoder.Encode(lastName),
+        GivenNameTraslitaerated = MrzEncoder.Encode(firstName),
```

Comment:
I appreciate this is how it was named in the previous DCC versions. This was updated in 1.3.0 to Surname instead of Family Name to match the EU regs. JSON field name stayed the same though (perhaps confusingly, but at least keeping the fieldname the same didn't break all the existing code)